### PR TITLE
update category unit model tests

### LINKED ISSUE #524: Write unit tests for the Category model
Write unit tests for the [`Category` Model](https://github.com/code-corps/code-corps-ember/blob/develop/app/models/category.js). There is an [existing unit test file](https://github.com/code-corps/code-corps-ember/blob/develop/tests/unit/models/category-test.js), but it could use more coverage (computed properties & relationship declarations). 

## References
- Progress on: #98 
- [Model testing guide](https://guides.emberjs.com/v2.8.0/testing/testing-models/)
- [Contributing Guidelines](https://github.com/code-corps/code-corps-ember/blob/develop/.github/CONTRIBUTING.md)
- Join our [Slack](https://codecorps.slack.com)

### PR DESCRIPTION
**Pull request by WenInCode**
# What's in this PR?
Update `category` unit model tests

## References
- Fixes #524 
- Progress on: #98

### CODE REVIEW

**Comment by JoshSmith** [comment]
Do you not want to test the attributes of the `Category` model here, too?

**Comment by WenInCode** [comment]
@joshsmith just following the ember model testing guide, my understanding is that there is value in testing functions, computer properties and relationship declarations. I'm not sure how much value there is in testing plain attributes... Unless there is validations on them

**Comment by begedin** [comment]
Same as https://github.com/code-corps/code-corps-ember/pull/547#issuecomment-250958697

It would make sense to add a test that ensures all attributes are accounted for.

**Comment by WenInCode** [comment]
I've updated this to test attribute existence. Reference [this](https://github.com/code-corps/code-corps-ember/pull/547#issuecomment-250982294) for method reasoning.

**Comment by WenInCode** [comment]
Going to update this with relationship helpers from #547 once they get merged.

File: tests/unit/models/category-test.js [hunk 1]
```diff
@@ -10,3 +11,17 @@ test('it exists', function(assert) {
...
+
+test('it should have all its attributes', function(assert) {
+  assert.expect(3);
+
+  const category = this.subject();
+  const attributes = Object.keys(category.toJSON());
+
+  assert.ok(attributes.includes('description'), 'should have description attribute');
+  assert.ok(attributes.includes('name'), 'should have name attribute');
+  assert.ok(attributes.includes('slug'), 'should have slug attribute');
```
Comment (JoshSmith) [inline tests/unit/models/category-test.js]: We should refactor these into an `assert.hasAttribute(attributes, 'attribute'`) which itself just runs `assert.ok()` with the description we use there.

Let's make an issue to circle back there. Will DRY these up slightly.

**Pull request merged**